src/test/java/seedu/address/logic/commands/TagCommandIntegrationTest.java

```diff
@@ -32,14 +32,14 @@ public void execute_newTag_success() {
         Model expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
         expectedModel.addTag(validPerson);
 
-        assertCommandSuccess(new AddCommand(validPerson), model,
-                String.format(AddCommand.MESSAGE_SUCCESS, validPerson), expectedModel);
+        assertCommandSuccess(new TagCommand(validPerson), model,
+                String.format(TagCommand.MESSAGE_SUCCESS, validPerson), expectedModel);
     }
 
     @Test
     public void execute_duplicateTag_throwsCommandException() {
         Tag personInList = model.getAddressBook().getTagList().get(0);
-        assertCommandFailure(new AddCommand(personInList), model, AddCommand.MESSAGE_DUPLICATE_TAG);
+        assertCommandFailure(new TagCommand(personInList), model, TagCommand.MESSAGE_DUPLICATE_TAG);
```

Comment:
Shouldnt personInList be tagInList instead?

Reply:
Roger that